services/web/server/src/simcore_service_webserver/workspaces/_models.py

```diff
@@ -46,6 +48,14 @@ class WorkspacesFilters(Filters):
         default=False,
         description="Set to true to list trashed, false to list non-trashed (default), None to list all",
     )
+    text: Annotated[
+        str | None, BeforeValidator(empty_str_to_none_pre_validator)
+    ] = Field(
+        default=None,
+        description="Multi column full text search",
```

Comment:
which column? Shouldn you add that they are in `name` OR `description`?  Why don't you create a template message and use it in all `text` search?